src/components/miscellaneous/MarkdownRenderer.tsx

```diff
@@ -34,37 +41,58 @@ const useStyles = makeStyles((theme) => ({
   list: {
     listStylePosition: 'inside',
   },
-  text: {
+  content: {
     marginBottom: theme.spacing(1),
   },
+  expansion: {
+    border: theme.palette.sizes.border.width + ' solid ' + theme.palette.colors.border.main,
+    background: theme.palette.colors.background.smoke,
+  },
 }));
 
 export type MarkdownRendererProps = {
   value: string;
 };
 
-function MarkdownRenderer({ value }: MarkdownRendererProps) {
+const MarkdownRenderer = ({ value }: MarkdownRendererProps) => {
   const classes = useStyles();
-  const getHeadingLevel = (level: number) => {
-    switch (level) {
-      case 1:
-        return 'h2';
-      default:
-        return 'h3';
-    }
+  const componentTransforms = {
+    Content: ({ children }: { children?: React.ReactNode }) => <div className={classes.content}>{children}</div>,
+    Expansion: ({ header = '', text }: { header?: string; text?: string; children?: React.ReactNode }) => (
+      <Expansion className={classes.expansion} flat header={header} text={text} />
+    ),
+    Event: ({ id }: { id?: string; children?: React.ReactNode }) => {
+      const [data] = useEventById(Number(id));
+      return data ? <ListItem event={data} largeImg /> : <></>;
```

Comment:
litt usikker på om det går, men tror det blir mer riktig å skrive 
```suggestion
      return data ? <ListItem event={data} largeImg /> : null;
``'